java/system-test/src/main/java/sleeper/systemtest/ingest/WaitForIngestTasks.java

```diff
@@ -54,15 +55,22 @@ public class WaitForIngestTasks {
             TASKS_FINISHED_POLL_INTERVAL_MILLIS, TASKS_FINISHED_TIMEOUT_MILLIS);
 
     public WaitForIngestTasks(
-            SystemTestProperties systemTestProperties,
+            InstanceProperties instanceProperties,
             AmazonSQS sqsClient,
             IngestTaskStatusStore taskStatusStore) {
         this.taskStatusStore = taskStatusStore;
         this.waitForEmptyQueue = WaitForQueueEstimate.isEmpty(
-                withSqsClient(sqsClient), systemTestProperties, INGEST_JOB_QUEUE_URL,
+                withSqsClient(sqsClient), instanceProperties, INGEST_JOB_QUEUE_URL,
                 PollWithRetries.intervalAndPollingTimeout(QUEUE_EMPTY_POLL_INTERVAL_MILLIS, QUEUE_EMPTY_TIMEOUT_MILLIS));
     }
 
+    public WaitForIngestTasks(
+            SystemTestProperties systemTestProperties,
+            AmazonSQS sqsClient,
+            IngestTaskStatusStore taskStatusStore) {
+        this((InstanceProperties) systemTestProperties, sqsClient, taskStatusStore);
+    }
```

Comment:
The second constructor isn't needed, since an instance of SystemTestProperties is also an instance of InstanceProperties.